### PR TITLE
Add generalised styling function

### PR DESCRIPTION
**Pull request by fenjalien**
#57

### CODE REVIEW

File: tests/run [hunk 1]
```diff
@@ -71,7 +71,7 @@ function run_test()
   )
 }
 
-install_typst
+# install_typst
```
Comment (johannes-wolf) [inline tests/run:74]: Is this problematic on Windows?

**Review by johannes-wolf — APPROVED** [review]
Looks really good!

File: tests/run (hunk 1 above)
Comment (fenjalien) [reply tests/run]: :/ Kind of, it got annoying that it had to install typst every time. Theres a few other changes I have to make to it to get it to work. Too be honest we need to overhaul the whole testing system.

**Review by johannes-wolf — APPROVED** [review]
LGTM

**Comment by johannes-wolf** [comment]
Looks like the ubuntu(?) CI does not install imgage-magick under the `magick` prefix binary?
We should test for this case in the tests script.

**Comment by fenjalien** [comment]
Right yea, testing isn't cross platform at the moment so I had to make some changes for my windows machine. I'll revert them

**Pull request merged**